stitch_assets/code/Chat_Interface_Screen.html

```
@@ -0,0 +1,226 @@
+    </style>
+<style>
+    body {
+      min-height: max(884px, 100dvh);
+    }
+  </style>
+  </head>
+<body class="bg-background font-body text-on-background min-h-screen flex flex-col">
+<!-- TopAppBar -->
+<header class="fixed top-0 w-full z-50 flex justify-between items-center px-6 py-3 w-full bg-indigo-50/80 dark:bg-slate-900/80 backdrop-blur-xl shadow-sm dark:shadow-none no-border tonal-shift">
+<div class="flex items-center gap-3">
+<button class="p-2 -ml-2 hover:bg-indigo-100/50 rounded-full transition-colors active:scale-95 duration-200 text-indigo-600">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)